src/main/java/io/paradoxical/dropwizard/swagger/bundles/SwaggerUIBundle.java

```diff
@@ -10,13 +11,17 @@
     private final SwaggerUIConfigurator swaggerUIConfigurator;
 
     public SwaggerUIBundle(@NonNull final SwaggerUIConfigurator swaggerUIConfigurator) {
-        super(SwaggerAssets.Assets);
+        this(SwaggerAssets.Assets, swaggerUIConfigurator);
+    }
+
+    public SwaggerUIBundle(@NonNull final AssetsDefinition assets, @NonNull final SwaggerUIConfigurator swaggerUIConfigurator) {
+        super(assets);
         this.swaggerUIConfigurator = swaggerUIConfigurator;
     }
 
     @Override
     public void run(final Environment environment) {
         super.run(environment);
-        swaggerUIConfigurator.configure(environment, environment.jersey().getResourceConfig());
+        swaggerUIConfigurator.configure(environment, environment.jersey());
```

Comment:
Is it even worth passing in environment.jersey if we already passing in environment? we're already pulling metrics off of it and some other things. not sure when we'd have a scenario to use a different jersey environment than the one attached to the dropwizard env